examples/testprograms/redis/main.star

```diff
@@ -0,0 +1,7 @@
+load("@redis", "myredis")
```

Comment:
System tests are better than examples, because they're executed automatically. I'll add/move this in a separate PR.